docs/_config.yml

```diff
@@ -36,6 +36,9 @@ search:
 # Supports true (default) or false
 heading_anchors: true
 
+# Set a path/url to a favicon that will be displayed by the browser
+favicon_ico: 'docs/img/modular-hero.ico'
```

Comment:
@sgb-io  there is a confusion on the relative path for fav icon. should i set it in context of project root or relative root ie. "/img/modular-hero.ico"?